static/js/components/capability/door-sensor.js

```diff
@@ -44,12 +44,13 @@ template.innerHTML = `
       font-weight: bold;
       text-transform: uppercase;
       padding-top: 8.75rem;
-      font-size: 1.4rem;
+      font-size: 1.2rem;
```

Comment:
Oh, sorry Mike. I think I made a mistake asking you to change this one. This string is short enough to stay at 1.4rem in open and closed states. Otherwise good to go!

Reply:
No problem. Fixed.